pkg/packagekit/autoupdates.jsx

```diff
@@ -203,16 +209,15 @@ class DnfImpl extends ImplBase {
 
 // Returns a promise for instantiating "backend"; this will never fail, if
 // automatic updates are not supported, backend will be null.
-function getBackend() {
-    if (!getBackend.promise) {
-        debug("getBackend() called first time, initializing promise");
+function getBackend(clearPromise) {
```

Comment:
`clearPromise` - took me a bit to understand what this does. Perhaps `forceReinit` or `noCache`?

Reply:
yea the name wasn't the best. noCache/forceReinit is a lot clearer. Will change.